pywbem/__init__.py

```diff
@@ -50,6 +51,17 @@
 
 from ._version import __version__  # noqa: F401
 
+# TODO: Verify whether pywbem needs a logging null handler.
+# import logging
+# try:  # Python 2.7+
```

Comment:
Change comment to # Python 2.7+ include NullHandler

Reply:
DONE